xtp/share/xtp/xml/localize.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<options>
+  <localize help="Allows you to compute localised orbitals using the Pipek-Mezey scheme">
```

Comment:
This is for the `localize` standalone tool? Should it not also have convergence limit as option, like in `dftgwbse.xml`?